steps/fabric-crashlytics-beta-deploy/2.3.3/step.yml

```diff
@@ -0,0 +1,119 @@
+title: Fabric/Crashlytics deployer
+summary: |-
+  Use the awesome Crashlytics/Fabric beta distribution service during your
+  workflow. Add notes, send notification to the testers, groups or to any email
+  address and let the search for those nasty bugs begin!
+description: |-
+  Deploys to Fabric/Crashlytics beta testing service.
+
+  ## How to setup your Xcode project for [Fabric](https://get.fabric.io/) / Crashlytics beta testing?
+
+  * Register a Fabric account at [https://get.fabric.io/](https://get.fabric.io/).
```

Comment:
Thank you for the comment. We will keep it in mind. We would like to merge this PR because it is a bit hard to release this step for cert issues.
We are working on the cert issue fix and we will take a look at the description and the step's source code to remove the Fabric related stuff.